pkg/web/router.go

```diff
@@ -27,14 +26,37 @@ import (
 	"github.com/canonical/identity-platform-admin-ui/pkg/status"
 )
 
-func NewRouter(idpConfig *idp.Config, schemasConfig *schemas.Config, rulesConfig *rules.Config, externalConfig ExternalClientsConfigInterface, wpool pool.WorkerPoolInterface, ollyConfig O11yConfigInterface) http.Handler {
+type RouterConfig struct {
+	idp                      *idp.Config
+	schemas                  *schemas.Config
+	rules                    *rules.Config
+	external                 ExternalClientsConfigInterface
+	payloadValidationEnabled bool
```

Comment:
we have an implicit convention of putting simpler types on top of the structure 

not a biggie anyway